pkg/controller/replicaset/replica_set.go

```diff
@@ -447,7 +447,7 @@ func (rsc *ReplicaSetController) manageReplicas(filteredPods []*api.Pod, rs *ext
 		for i := 0; i < diff; i++ {
 			go func() {
 				defer wait.Done()
-				if err := rsc.podControl.CreatePods(rs.Namespace, &rs.Spec.Template, rs); err != nil {
+				if err := rsc.podControl.CreatePods(rs.Namespace, &rs.Spec.Template, rs, false); err != nil {
```

Comment:
Yeah, the logic should be very similar, but probably separate PR. This one is large enough.